lib/model/bloc/startup/ChatResponseToPlotInfoTransformer.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:farmsmart_flutter/chat/ui/viewmodel/ChatResponseViewModel.dart';
+import 'package:farmsmart_flutter/model/bloc/Transformer.dart';
+import 'package:intl/intl.dart';
+
+class _Strings {
+  static const dateFormat = "MMMM";
+}
+
+class ChatResponseToPlotInfoTransformer extends ObjectTransformer<Map<String,ChatResponseViewModel>, Map<String,String>> {
+
+  final dateFormat = DateFormat(_Strings.dateFormat);
+
+  @override
+  Map<String,String> transform({Map<String,ChatResponseViewModel> from}) {
+    return from.map((key,value){
```

Comment:
Maybe change this alias naming to something like **responseViewModel** or **response** or **viewModel** just to avoid this kind of invocation 
**value.value**

Reply:
👍